tests/utils/tools.php

```diff
@@ -1,5 +1,8 @@
 <?php
 
+use MongoDB\BSON\DBPointer;
+use MongoDB\BSON\Symbol;
+use MongoDB\BSON\Undefined;
```

Comment:
The Undefined class isn't referenced anywhere.

```suggestion
```

I noted that we typically created instances of Undefined using Extended JSON. See: [`bson-undefined-serialization-002.phpt`](https://github.com/mongodb/mongo-php-driver/blob/v1.x/tests/bson/bson-undefined-serialization-002.phpt).

You already have working implementations for DBPointer and Symbol here, so probably not reason to touch those further. Thanks for parameterizing them.

Why not the same approach for DBPointer and Symbol?